lib/testmap.py

```diff
@@ -56,6 +56,7 @@
             'fedora-testing/dnf-copr',
             'rhel-8-6-distropkg',
             'rhel-9-0-distropkg',
+            'ubuntu-2204',
```

Comment:
Would be great, if these could land in separate PR so we can trigger at least starter kit to see if the image at least boots and has basic functionality. Or we could just trigger the new context manually.

Reply:
I triggered the test in https://github.com/cockpit-project/cockpit-podman/pull/947 already. I initially typo'ed the PR number, but still, it's running.